definitions/variable/industry/tag_paper_commodities.yaml

```diff
@@ -0,0 +1,7 @@
+- Pulp and Paper Commodity:
+    - Pulp:
+        description: pulp produced from pulpwood, mainly used for paper, board, textiles and cellophane
+        tier: "2"
+    - Paper:
+        description: think sheet material made from cellulose fibers
```

Comment:
Are you sure that "think sheet" is a thing?

Reply:
Ah this is a typo 😅 it should be "thin sheet" (borrowed from the first sentence on [wikipedia](https://en.wikipedia.org/wiki/Paper))